CHANGELOG.md

```diff
@@ -214,4 +215,4 @@ Please see the Release Notes for [v0.41.x](https://github.com/cosmos/cosmos-sdk/
 
 The IBC module was removed in the commit hash [da064e13d56add466548135739c5860a9f7ed842](https://github.com/cosmos/cosmos-sdk/commit/da064e13d56add466548135739c5860a9f7ed842) on the SDK. The release for SDK v0.43.0 will be the first release without the IBC module.
 
-Backports should be made to the [release/v0.42.x](https://github.com/cosmos/cosmos-sdk/tree/release/v0.42.x) branch on the SDK.
\ No newline at end of file
+Backports should be made to the [release/v0.42.x](https://github.com/cosmos/cosmos-sdk/tree/release/v0.42.x) branch on the SDK.
```

Comment:
@seantking will you also open a PR on the SDK? Let me know if you would like me to do instead.